src/routes/backingImage/BackupBackingImage.js

```diff
@@ -3,18 +3,32 @@ import PropTypes from 'prop-types'
 import { connect } from 'dva'
 import { Row, Col } from 'antd'
 import cx from 'classnames'
+import queryString from 'query-string'
+import { routerRedux } from 'dva/router'
 import { Filter } from '../../components/index'
 import BackupBackingImageList from './BackupBackingImageList'
 import BackupBackingImageBulkActions from './BackupBackingImageBulkActions'
 import RestoreBackupBackingImageModal from './RestoreBackupBackingImageModal'
 import { filterBackupBackingImage } from './utils'
 import styles from './BackupBackingImage.less'
 
-function BackupBackingImage({ backingImage, loading, dispatch, location, className }) {
+function BackupBackingImage({
+  backingImage,
+  loading,
+  dispatch,
+  location = {},
+  className,
+  persistFilterInURL = false
+}) {
+  const { pathname, hash, search } = location
+
   const containerRef = useRef(null)
   const actionBarRef = useRef(null)
   const [tableBodyHeight, setTableBodyHeight] = useState(0)
+  const [bbiSearchField, setBbiSearchField] = useState('')
+  const [bbiSearchValue, setBbiSearchValue] = useState('')
```

Comment:
Moved from redux to allow separate state management for the table, because it is reused across two pages (Backup and Backing Image pages).